src/views/Network.js

```diff
@@ -45,7 +45,7 @@ class Network extends React.Component {
     if (network === NETWORK_NAMES.LOCAL) {
       setNetworkType(network)
 
-      const endpoint = 'wss://localhost:8545'
+      const endpoint = 'ws://localhost:8545'
```

Comment:
I've seen weird behavior around this (both wss and ws). Maybe I'm just doing something wrong in my dev environment. Let's talk about that today. (But this change specifically is fine, probably.)